community/berry/build

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+make
+make DESTDIR="$1" install
```

Comment:
You need to use `make DESTDIR="$1" PREFIX=/usr install` so that the package gets installed to `/usr` instead of `/usr/local` which is used for packages installed outside the package manager.

Reply:
Ah great, thank you!